pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -171,6 +178,31 @@ func (o *MustGatherOptions) Complete(f kcmdutil.Factory, cmd *cobra.Command, arg
 	if len(o.DestDir) == 0 {
 		o.DestDir = fmt.Sprintf("must-gather.local.%06d", rand.Int63())
 	}
+	// TODO: this should be in Validate() method, but added here because of the call to o.completeImages() below
+	if o.AllImages {
+		errStr := fmt.Sprintf("and --all-images are mutually exclusive: please specify one or the other")
+		if len(o.Images) != 0 {
+			return fmt.Errorf("--image %s", errStr)
+		}
+		if o.HostNetwork {
+			return fmt.Errorf("--host-network %s", errStr)
+		}
+		if len(o.ImageStreams) != 0 {
+			return fmt.Errorf("--image-streams %s", errStr)
+		}
+		if o.NodeName != "" {
+			return fmt.Errorf("--node-name %s", errStr)
+		}
+		if o.RunNamespace != "" {
+			return fmt.Errorf("--run-namespace %s", errStr)
+		}
+		if o.SourceDir != "/must-gather/" {
+			return fmt.Errorf("--sourec-dir %s", errStr)
+		}
+		if o.VolumePercentage != 30 {
```

Comment:
Why don't we allow changing the volume percentage. On the contrary, I think we should allow it.

Reply:
OK, I'll remove this check.

Reply:
Removed.